### ISSUE TITLE
Tests aren't running in Actions

### ISSUE THREAD

**Issue by K-Meech** [mobie/mobie-io#54]
The tests in this repository aren't running during the build e.g. see: https://github.com/mobie/mobie-io/runs/4966733184?check_suite_focus=true#step:6:102 For some reason they're not being found by maven.

**Comment by K-Meech** [comment]
They also don't run locally with ```mvn clean install```

**Comment by tischi** [comment]
Probably we should include `develop` [here](https://github.com/mobie/mobie-io/blob/a6e8285fd2323ae316b9d1b8e96c643c4870b255/.github/workflows/build-main.yml#L4)?
Do you think that could be the issue?

**Comment by tischi** [comment]
Ah, sorry, on the `develop` branch it is fine, in fact: https://github.com/mobie/mobie-io/blob/5ff3e1b7d195fd3a28c4f856e5289bac59e2cd3f/.github/workflows/build-main.yml#L4

**Comment by K-Meech** [comment]
I think it's something to do with the dependencies in the pom - maybe the junit version vs maven version? (since it doesn't run locally either)

**Comment by K-Meech** [comment]
Adding this dependency seems to fix it:
```
<dependency>
            <groupId>org.junit.jupiter</groupId>
            <artifactId>junit-jupiter-engine</artifactId>
            <version>5.5.2</version>
            <scope>test</scope>
</dependency>
```
But some of the tests currently fail. I can add it to the current PR I'm working on.

**Comment by K-Meech** [comment]
I added it here: https://github.com/mobie/mobie-io/pull/55

**Comment by constantinpape** [comment]
Solved by #55
